README.md

```diff
@@ -1,35 +1,38 @@
 [![Build Status](https://travis-ci.org/splunk/splunk-sdk-javascript.svg?branch=master)](https://travis-ci.org/splunk/splunk-sdk-javascript)
-# The Splunk Software Development Kit for JavaScript
+# The Splunk Enterprise Software Development Kit for JavaScript
 
-#### Version 1.9.0
+#### Version 1.9.1
 
-The Splunk Software Development Kit (SDK) for JavaScript contains library code and
-examples designed to enable developers to build applications using Splunk and
-JavaScript. This SDK supports both server- and client-side JavaScript.
+The Splunk Enterprise Software Development Kit (SDK) for JavaScript contains library code and examples designed to enable developers to build applications using the Splunk platform and JavaScript. This SDK supports server-side and client-side JavaScript.
+
+For more information, see [Splunk Enterprise SDK for JavaScript](https://dev.splunk.com/enterprise/docs/devtools/javascript/sdk-javascript/) on the Splunk Developer Portal.
 
 ## Requirements
 
-* Node.js v.0.12, or v4 or later. The Splunk SDK for Javascript is tested with Node.js v.0.12, v4.2, and v10.0. 
-* Splunk Enterprise 6.3.0 or later, or Splunk Cloud. The Splunk SDK for Javascript is tested with Splunk Enterprise 7.0 and 7.2. 
+* Node.js v 0.12, or v4 or later
```

Comment:
Since we updated the test matrix let's update the README sections indicating compatibility as well - this should be whatever our matrix uses, so I think `v8.17.0, or v14 or later` assuming we add 8.17.0 per comment above

Reply:
README.md updated